manifests/eksctl-config-cluster2.yaml

```diff
@@ -1,13 +1,25 @@
 apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 availabilityZones:
-  - us-east-1c
-  - us-east-1d
+  - ${CLUSTER_REGION}a
```

Comment:
Since we ask the user to check which availability zones are ideal for them, can we parametrize this so that they can define it, rather than us adding 'a,b' or whatever static AZs we're picking here?